mappings/GameTest.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_6302 GameTest
```

Comment:
Hmm, actually `GameTest` is bad because it's a duplicate name. Also you need to choose a package

Reply:
Duplicate name? The old `GameTest` is `GameTestState` now so I don't think there shouldn't be any duplicates 🤔

Reply:
I think it moved it out of the package because the name was duplicate for a moment for some reason (even though I renamed the other `GameTest` before I renamed the `TestImplementation`). Moved it back there.